test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -15,26 +16,131 @@ var customMatchers = require('../assets/custom_matchers');
  *
  */
 
+var PLOT_DELAY = 200;
+var MOUSE_DELAY = 20;
+var MODEBAR_DELAY = 500;
 
-describe('Test plot structure', function() {
+
+describe('Test gl plot interactions', function() {
     'use strict';
 
+    var gd;
+
     beforeEach(function() {
         jasmine.addMatchers(customMatchers);
     });
 
-    afterEach(destroyGraphDiv);
+    afterEach(function() {
+        var fullLayout = gd._fullLayout,
+            sceneIds;
+
+        sceneIds = Plots.getSubplotIds(fullLayout, 'gl3d');
+        sceneIds.forEach(function(id) {
+            fullLayout[id]._scene.destroy();
```

Comment:
Now, each gl context is destroyed after each gl test run, making for more consistent test runs.